packages/api-explorer/src/index.jsx

```diff
@@ -209,6 +209,11 @@ class ApiExplorer extends React.Component {
     const NewGlossaryTermsContext = utils.GlossaryContext;
     /* eslint-enable global-require */
 
+    const onAuthGroupChange = auth => {
+      if (this.props && this.props.onAuthGroupChange) this.props.onAuthGroupChange(auth);
```

Comment:
Might want to check if this is `if (this.props && typeof this.props.onAuthGroupChange === 'function')` to continue `proptype` restrictions and ensure it's callable.